README.md

```diff
@@ -20,26 +25,36 @@ What this means is that unlike many other Prettier plugins,
 `prettier-java` has **no additional runtime pre-requisites** (e.g: Python executable).
 It could even be used inside a browser.
 
+## Subpackages
+
+This project contains 2 packages:
+
+- [java-parser](./packages/java-parser) A Java Parser using [Chevrotain](https://github.com/SAP/chevrotain) which output a **C**oncrete **S**yntax **T**ree
+- [prettier-plugin-java](./packages/prettier-plugin-java) A plugin for Prettier to format Java code
```

Comment:
List the prettier-plugin-java package first.
Most people who come to this repo are interested in the prettier plugin not its implementation details